src/webui/src/components/Sessionpro.tsx

```diff
@@ -88,6 +88,20 @@ class Sessionpro extends React.Component<{}, SessionState> {
         };
     }
 
+    convertTime = (num: number) => {
+        let hour = Math.floor(num / 3600 % 24);
+        let result: string = '';
+        if (num % 3600 === 0) {
+            return result = hour + 'h';
+        } 
+        let min = Math.floor(num / 60 % 60);
+        if (hour < 1) {
+            return result = min + '';
+        } else {
+            return result = hour + 'h  ' + min + 'min';
+        }
+    }
```

Comment:
This function can be simplified as following:
const hour =  Math.floor(num / 3600);
const min = Math.floor(num / 60 % 60);
return hour > 0 ? `${hour} h ${min} min` : `${min} min`

Reply:
Thanks very much.